core/domain/feedback_services_test.py

```diff
@@ -782,15 +781,14 @@ def test_create_new_instant_task(self):
         feedback_services.enqueue_feedback_message_instant_email_task(
             user_id, reference)
         self.assertEqual(
-            self.count_jobs_in_taskqueue(taskqueue_services.QUEUE_NAME_EMAILS),
+            self.count_jobs_in_taskqueue(
+                taskqueue_services.QUEUE_NAME_EMAILS),
             1)
 
         tasks = self.get_pending_tasks(
             queue_name=taskqueue_services.QUEUE_NAME_EMAILS)
-        payload = json.loads(tasks[0].payload)
```

Comment:
Why isn't this replicable?

Reply:
Fixed, I removed it as part of an earlier iteration.